p4c.bzl

```diff
@@ -110,5 +126,14 @@ run_p4c = rule(
             mandatory = False,
             default = "",
         ),
+        "p4runtime_files": attr.output_list(
+            doc = """
+                File(s) path(s) to write the control plane
+                API description (P4Info) to. Acceptable extensions
+                are ".txt", ".json", and ".bin".
+                """,
+            mandatory = False,
+            allow_empty = True,
```

Comment:
allow_files = [".txt", ".json", ".bin"]

Reply:
apparently this does not work for output files! https://docs.bazel.build/versions/master/skylark/lib/attr.html#output_list

Reply:
Oh, interesting :)